### PR TITLE
Make jnp.gradient() call signature match np.gradient

### PR DESCRIPTION
**Pull request by jakevdp**

### CODE REVIEW

File: tests/lax_numpy_test.py [hunk 1]
```diff
@@ -4497,7 +4497,6 @@ def testWrappedSignaturesMatch(self):
       'eye': ['order'],
       'full': ['order'],
       'full_like': ['subok', 'order'],
-      'gradient': ['varargs', 'axis', 'edge_order'],
```
Comment (froystig) [inline tests/lax_numpy_test.py:4500]: Did `axis` become unsupported in this change?

**Review by froystig — APPROVED** [review]

File: tests/lax_numpy_test.py (hunk 1 above)
Comment (jakevdp) [reply tests/lax_numpy_test.py:4500]: axis is still supported, but now is supported as an explicit keyword, rather than via ``**kwargs``

Comment (froystig) [reply tests/lax_numpy_test.py:4500]: I misread the diff! Got it now.
